swim/join_test.go

```diff
@@ -92,20 +95,20 @@ func (s *JoinSenderTestSuite) TestSelectMultipleGroups() {
 }
 
 func (s *JoinSenderTestSuite) TestSelectGroupExcludes() {
-	seedBootstrapHosts(s.node, genAddresses(1, 1, 5))
+	seedBootstrapHosts(s.node, fakeAddresses(1, 1, 5))
 
 	joiner, err := newJoinSender(s.node, nil)
 	s.Require().NoError(err, "cannot have an error")
 	s.Require().NotNil(joiner, "joiner cannot be nil")
 
-	group := sort.StringSlice(joiner.SelectGroup([]string{"127.0.0.1:3003", "127.0.0.1:3005"}))
+	group := sort.StringSlice(joiner.SelectGroup([]string{"192.0.2.1:3", "192.0.2.1:5"}))
 	group.Sort()
 
-	s.EqualValues([]string{"127.0.0.1:3002", "127.0.0.1:3004"}, group)
+	s.EqualValues([]string{"192.0.2.1:2", "192.0.2.1:4"}, group)
```

Comment:
Can you guys suggest an appropriate comment? Should we add this comment to *all* lines that explicitly use this address as a test expectation? Maybe we just add this comment to line 42 at the top?